app/controllers/api/v2/participant_declarations_controller.rb

```diff
@@ -49,8 +49,12 @@ def show
       def query_scope
         scope = ParticipantDeclaration.for_lead_provider(cpd_lead_provider)
         scope = scope.where("user_id = ?", participant_id_filter) if participant_id_filter.present?
-        scope = scope.where("updated_at > ?", updated_since) if updated_since.present?
-        scope
+
+        if updated_since.present?
+          scope.where("updated_at > ?", updated_since)
```

Comment:
`scope` needs to be mutated here otherwise as it stands this line won't do anything